packages/pilot/public/locales/pt/translations.json

```
@@ -942,10 +943,13 @@
       "wrong_password": "Senha inválida"
     },
     "reprocess": {
-      "submit": "Reprocessar transação",
       "confirmation": "Confirmação",
+      "failure": "Algo deu errado e não foi possível reprocessar essa transação.",
       "result": "Conclusão",
-      "success": "A transação foi reprocessada com sucesso",
+      "submit": "Reprocessar transação",
+      "success_reprocess_disclaimer": "Se você utiliza algum serviço ou plataforma, lembre-se de atualizar o status dessa transação.",
+      "success_with_antifraud": "A transação reprocessada com antifraude foi realizada com sucesso",
+      "success_without_antifraud": "A transação reprocessada sem antifraude foi realizada com sucesso!",
```

Gente, validamos o texto aqui e chegamos em:
```suggestion
      "success_without_antifraud": "A transação foi reprocessada sem antifraude com sucesso.",
```

---

Gente, discutimos os dois textos (success_with_antifraud e success_without_antifraud) e acreditamos que faça sentido simplificarmos o texto para `A transação foi reprocessada com sucesso.`.

---

Boa! @0liveira, corrigido :slightly_smiling_face: